src/vivarium_gates_nutrition_optimization_child/model_specifications/nutrition_optimization_child.yaml

```diff
@@ -119,6 +119,7 @@ configuration:
     stratification:
         default:
             - 'age_group'
```

Comment:
Do we know what age groups these will be? We use custom age groups that are different than the GBD age groups for production runs, but it would be most convenient to have GBD age groups for V&V.

Reply:
Currently these are the age groups that results could be stratified by. This looks correct to me and like it is not the production run 0-6 months, 6-18 etc. But let me know if you have concerns! 

![image](https://github.com/ihmeuw/vivarium_gates_nutrition_optimization_child/assets/108029462/4ca65edf-fa29-41bf-ab66-d53f262d7830)

Reply:
Ok perfect! Thank you!